cli/tests/test_operations.rs

```diff
@@ -738,7 +738,7 @@ fn test_op_recover_from_bad_gc() {
     let repo_path = test_env.env_root().join("repo");
     let git_object_path = |hex: &str| {
         let (shard, file_name) = hex.split_at(2);
-        let mut file_path = repo_path.to_owned();
+        let mut file_path = repo_path.clone();
```

Comment:
I'm not against this lint rule, but I tend to use `.to_owned()` if the owned type has a corresponding slice type, and an owned type isn't needed at call site.

Reply:
The idea behind this lint is that sometimes you may remove the `.clone()` completely as you already have the type you want. When you call `.to_owned()`, it does not appear as clearly, and you are not warned by the compiler or clippy about it.

Reply:
I've made them warnings.